src/OSPSuite.UI/Views/ObservedData/DataRepositoryDataView.cs

```diff
@@ -1,79 +1,37 @@
 ﻿using System.Data;
 using System.Linq;
-using OSPSuite.DataBinding.DevExpress.XtraGrid;
-using OSPSuite.Utility.Extensions;
 using DevExpress.XtraEditors.Controls;
-using DevExpress.XtraEditors.Repository;
-using DevExpress.XtraGrid.Columns;
-using DevExpress.XtraGrid.Views.Base;
-using DevExpress.XtraLayout.Utils;
-using OSPSuite.Core.Commands;
-using OSPSuite.Presentation.DTO;
 using OSPSuite.Presentation.Presenters.ObservedData;
 using OSPSuite.Presentation.Views.ObservedData;
 using OSPSuite.UI.Binders;
-using OSPSuite.UI.Controls;
 using OSPSuite.UI.Services;
 
 namespace OSPSuite.UI.Views.ObservedData
 {
-   public partial class DataRepositoryDataView : BaseDataRepositoryDataView<IDataRepositoryDataView, IDataRepositoryDataPresenter>, IDataRepositoryDataView
+   public partial class DataRepositoryDataView : BaseDataRepositoryDataView<IDataRepositoryDataView, IDataRepositoryDataPresenter>,
+      IDataRepositoryDataView
    {
       protected readonly GridViewColumnUnitsMenuBinder<int> _columnUnitsMenuBinder;
-      private readonly RepositoryItemButtonEdit _removeButtonRepository = new UxRemoveButtonRepository();
-      private readonly GridColumnCreator _creator = new GridColumnCreator();
       private DataTable _dataTable;
-      private GridColumn _removeColumn;
-      private bool _editable = true;
 
       public DataRepositoryDataView(IToolTipCreator tooltipCreator) : base(tooltipCreator)
       {
          InitializeComponent();
          _columnUnitsMenuBinder = new GridViewColumnUnitsMenuBinder<int>(gridView, col => col.AbsoluteIndex);
-      }
-
-      public void DisableEdition()
-      {
-         btnAddData.Enabled = false;
-         layoutItemAddData.Visibility = LayoutVisibility.Never;
-         emptySpaceItem.Visibility = LayoutVisibility.Never;
          gridView.OptionsBehavior.Editable = false;
-         if (_removeColumn != null)
-         {
-            gridView.Columns.Remove(_removeColumn);
-         }
-         gridView.OptionsView.ShowIndicator = false;
-
-         _editable = false;
       }
 
       public override void InitializeBinding()
       {
          base.InitializeBinding();
          gridView.ValidatingEditor += (o, e) => OnEvent(gridEditorValidating, e);
-         gridView.CellValueChanged += (o, e) => OnEvent(gridViewOnCellValueChanged, e);
-         _removeButtonRepository.ButtonClick += (o, e) => OnEvent(_presenter.RemoveData, gridView.GetDataSourceRowIndex(gridView.FocusedRowHandle));
-         btnAddData.Click += (o, e) => OnEvent(_presenter.AddRow);
       }
 
       public override void BindTo(DataTable dataTable)
       {
          _dataTable = dataTable;
          base.BindTo(dataTable);
          _columnUnitsMenuBinder.BindTo(_presenter);
```

Comment:
Do we need this?

Reply:
@msevestre are you refering to the ValidatingEditor? If yes, then I agree it is not necessary. I am checking a bit if everything still works when removing all the code

Reply:
removed